phpstan-baseline.neon

```diff
@@ -1,2 +1,6 @@
 parameters:
-	ignoreErrors: []
+	ignoreErrors:
+		-
+			message: "#^Method TTN\\\\Tea\\\\Domain\\\\Repository\\\\TeaRepository\\:\\:findByOwnerUid\\(\\) return type with generic interface TYPO3\\\\CMS\\\\Extbase\\\\Persistence\\\\QueryResultInterface does not specify its types\\: ModelType$#"
```

Comment:
I don't understand why we need this, can you explain?
I'd expect saschaegerer/phpstan-typo3 to take care of this. Furthermore we annotated the repository: https://github.com/FriendsOfTYPO3/tea/blob/main/Classes/Domain/Repository/TeaRepository.php#L13

Reply:
We probably need to add a return type annotation using the generic for this method. (As far as I understand, saschaegerer/phpstan-typo3 only works for the methods that the repository base class already has.)